src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -844,6 +867,18 @@ def deviceAvailable(self, device, err):
 
         return DeviceProxyWrapper(returnDevice, self._dmLib)
 
+    async def WaitForActive(self, nodeid, stayActiveDurationMs=30000):
+        ''' Waits a LIT ICD device to become active. Will send a StayActive command to the device on active to allow human operations.
+
+        nodeId: Node ID of the LID ICD
+        stayActiveDurationMs: The duration in the StayActive command, in milliseconds
+
+        Returns:
+            - StayActiveResponse on success
+        '''
+        await icd.WaitForCheckIn(self._fabricIndex, nodeid)
```

Comment:
This could pend for a really really long time. There needs to be a time-out for this overall API